lib/yelp/fusion/responses/models/hours.rb

```diff
@@ -38,4 +38,4 @@ def initialize(json)
       end
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
Sorry about this line change, it is automatic through my editor. Feel free to ignore/drop it. 🙇🏻‍♂️